go/host/host.go

```diff
@@ -889,7 +900,7 @@ func (h *host) bootstrapHost() types.Block {
 		result, err := h.enclaveClient.SubmitBlock(cb, false)
 		if err != nil {
 			var bsErr *common.BlockRejectError
-			isBSE := errors.As(err, bsErr)
+			isBSE := errors.As(err, &bsErr)
```

Comment:
isn't this the address of the pointer?

Reply:
Yeah I found it weird too but actually this was a bug here before and if you look at the usage in online examples it always seems to be done this way, I sent some links on discord including this example from official docs:

<img width="234" alt="Screenshot 2022-11-17 at 10 37 50" src="https://user-images.githubusercontent.com/98158711/202424524-b6ec795c-4c61-4e9d-8daf-87a4b2110d28.png">